classes/db/DbPDO.php

```diff
@@ -464,6 +469,18 @@ public static function tryUTF8($server, $user, $pwd)
         return ($result === false) ? false : true;
     }
 
+    /**
+     * Sets UTF-8 to given connection.
+     *
+     * @param PDO $link connection instance
+     *
+     * @return bool if setting UTF-8 charset was successful
+     */
+    public static function setCharsetUTF8($link)
+    {
+        return ($link->query('SET NAMES \'utf8\'') === false) ? false : true;
```

Comment:
ternary condition is useless.

`$link->query('SET NAMES \'utf8\'')` or if you want to be sureit is a bool `(bool) $link->query('SET NAMES \'utf8\'')`

Reply:
Agree. But leaving !== false should be kept I think. It keeps it's behavior